compiler/AST/expr.cpp

```diff
@@ -4468,6 +4469,8 @@ GenRet CallExpr::codegen() {
     case PRIM_ARRAY_GET:
     case PRIM_ARRAY_GET_VALUE:
     case PRIM_ON_LOCALE_NUM:
+    case PRIM_GET_REAL:
+    case PRIM_GET_IMAG:
       // generated during generation of PRIM_MOVE
```

Comment:
I updated the PR message. It's a bug fix for a bug I discovered in earlier versions of this PR.